devito/types/relational.py

```diff
@@ -36,9 +36,8 @@ class Le(sympy.Le):
     >>> g = Function(name='g', shape=grid.shape, dimensions=grid.dimensions)
     >>> x, y = grid.dimensions
     >>> cond = Le(g, 1)
-    >>> ci = ConditionalDimension(name='ci', parent=y, condition=cond)
-    >>> f = Function(name='f', shape=grid.shape, dimensions=(x, ci))
-    >>> op = Operator(Eq(g, f))
+    >>> cond
```

Comment:
last nitpicking, ca you also add the `>>cond._conditon` so that can see the indexed expression is shifted correctly?

Reply:
Yes, funny it is that I was about to ask that.